public/css/styles.css

```diff
@@ -8772,6 +8782,11 @@ https://flickity.metafizzy.co
   cursor: pointer;
 }
 
+.icon-warning {
+  width: 14px;
+  height: 14px;
+  background-image: url(/assets/images/warning-sign.svg); }
```

Comment:
I used the existing warning .svg - however, the one shown in #385 design/screenshot looks different - do we have it somewhere as .svg file as well ?

Reply:
@jholdstock - maybe you can navigate here ?

Reply:
Added new style icon as well